src/diffpy/utils/parsers/loadmetafile.py

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env python
+##############################################################################
+#
+# diffpy.utils      by DANSE Diffraction group
+#                   Simon J. L. Billinge
+#                   (c) 2010 The Trustees of Columbia University
+#                   in the City of New York.  All rights reserved.
+#
+# File coded by:
+#
+# See AUTHORS.txt for a list of people who contributed.
+# See LICENSE_DANSE.txt for license information.
+#
+##############################################################################
+
+import pathlib
+import json
+
+# FIXME: add support for yaml, xml
+supported_formats = ['.json']
+
+
+def load_PDF_into_db(dbname, pdfname, hddata: dict, rv: list, show_path=True):
```

Comment:
this name is a bit confusing.  "load" using means "import into memory", like `json.load`.   Can you think carefully what this function does and give it a name that captures that clearly?  Also, update the docstring accordingly.

you talk about "database file" but such a thing doesn't exist.  Do you mean a json format file?

We often write clients that use CRUD commands and the client can be changed depending on the backend.  If the backend is a file-syste, we select a file-system client and write a json file.  IF the backend is a mongoDB database we select the mongodb client and then write code that connects to the database and inserts the information and so on.  That may be too complicated for what we want to do here, but make the docstrings clear so that a future programmer writing a more fancy client manager can immediately figure out what is happening and how to fix it up.  i.e., don't mention "database" anywhere in this function.

Reply:
Got it, made it serialize instead since it refers specifically to serial languages like json, xml, and yaml.